hapi-loader/pom.xml

```diff
@@ -10,10 +10,23 @@
         <finalName>${project.artifactId}</finalName>
     </build>
 
+    <repositories>
```

Comment:
Why is this needed?

Reply:
to tell maven to fetch from repositories other than maven.org